src/main/java/thePackmaster/packs/PickThemAllPack.java

```diff
@@ -0,0 +1,37 @@
+package thePackmaster.packs;
+
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.localization.UIStrings;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.cards.pickthemallpack.*;
+
+import java.util.ArrayList;
+
+public class PickThemAllPack extends AbstractCardPack {
+    public static final String ID = SpireAnniversary5Mod.makeID("PickThemAllPack");
+    private static final UIStrings UI_STRINGS = CardCrawlGame.languagePack.getUIString(ID);
+    public static final String NAME = UI_STRINGS.TEXT[0];
+    public static final String DESC = UI_STRINGS.TEXT[1];
+    public static final String AUTHOR = UI_STRINGS.TEXT[2];
+    public static final String CREDITS = UI_STRINGS.TEXT[3];
+
+    public PickThemAllPack() {
+        super(ID, NAME, DESC, AUTHOR, CREDITS, new PackSummary(3, 3, 4, 3, 4));
```

Comment:
This pack has a fair amount of variance, but calling it average offense/defense/frontload and above average support and scaling (various ways of improving your deck and building up big effects) seemed right from my playtesting.